app/uk/gov/hmrc/hmrcemailrenderer/templates/eeitt/EeittTemplates.scala

```diff
@@ -1467,6 +1467,15 @@ object EeittTemplates {
       plainTemplate = txt.qahcCode.f,
       htmlTemplate = html.qahcCode.f,
       priority = Some(MessagePriority.Urgent)
-    )
+    ),
+    MessageTemplate.create(
+      templateId = "pods_confirmation",
+      fromAddress = govUkTeamAddress,
+      service = Eeitt,
+      subject = "You requested a pension scheme tax refund from HMRC",
+      plainTemplate = txt.podsConfirmation.f,
+      htmlTemplate = html.podsConfirmation.f,
+      priority = Some(MessagePriority.Standard)
```

Comment:
Please remove this comma for last item , build will fail in jenkis because of scala check plugin

Reply:
@muralishub 

I've removed the comma and pushed again. Just waiting for the check to go green :)